pom.xml

```diff
@@ -13,17 +13,12 @@
   <description>Adds the ability to wrap your job with pre- and post- build steps which are only executed when a manual release build is triggered.</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Release+Plugin</url>
 
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5.1</version>
-        <configuration>
-          <goals>deploy</goals>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <properties>
+    <jenkins.version>1.609.3</jenkins.version>
```

Comment:
No, it's common sense. 1.481 was not even Jenkins, and 1.609.3 is the oldest LTS in a 1 year window. So I think this bump is not going to affect anyone.